changelog/unreleased/pull-2718

```diff
@@ -0,0 +1,17 @@
+Enhancement: Improve pruning performance and make pruning more customizable
+
+The prune command is now much faster. This is especially the case for remote
+repositories or repositories with not much to prune.
+Also the memory usage of the prune command is now reduced.
+
+The prune command now also supports new user flags, see
+https://restic.readthedocs.io/en/stable/060_forget.html
+
+Fixes several open issues, e.g.:
+https://github.com/restic/restic/issues/1599
+https://github.com/restic/restic/issues/1985
+https://github.com/restic/restic/issues/2162
```

Comment:
#2162 is mostly about parallelizing prune, which is not in scope for this PR.

Reply:
removed.